frontend/src/components/table/Table.js

```diff
@@ -375,6 +409,12 @@ class Table extends Component {
       const { selected } = this.state;
 
       if (tabInfo) {
+        dispatch(
+          updateTableSelection({
+            tableId: getTableId({ windowType: windowId, viewId }),
+            ids: selected,
+          })
+        );
         dispatch(
           selectTableItems({
```

Comment:
why do we have `selectTableItems` along with `updateTableSelection`?

Reply:
We're adding the new way of handling selections to work alongside the previous solution. This way we don't introduce breaking changes and can potentially roll it out anytime, and the old code will be removed completely with https://github.com/metasfresh/me03/issues/4539

Reply:
@siemiatj  @petrican  Ok, then I would suggest adding a TODO/FIXME which mentions that will be deleted on <which task>.

In this way we make sure we won't forget about it and it's also transparent for reviewer what's the intention.